src/content/plugins/internal-plugins.md

```diff
@@ -312,14 +312,6 @@ Merges chunks until each chunk has the minimum size of `minChunkSize`.
 
 Adds chunk ids of chunks which are included in the chunk. This eliminates unnecessary chunk loads.
 
-### UglifyJsPlugin
```

Comment:
why remove and not replace with TerserPlugin mention ?

Reply:
This page is about internal webpack plugins (ie plugins in the webpack repository) - which `UglifyJsPlugin` used to be, but was since moved out. As such, the `UglifyJsPlugin` this section is referring to actually doesn't exist any more:
https://github.com/webpack/webpack/blob/2f78aae8d8f78ce9c9221ff25a0483d81809bd81/lib/webpack.js#L179-L184

Reply:
ah true, thx for